docker/docker-compose.yaml

```diff
@@ -28,7 +28,7 @@ services:
 
   test:
     <<: *common
-    command: /bin/bash -xcl "swift test -Xswiftc -warnings-as-errors $${SANITIZER_ARG-}"
+    command: /bin/bash -xcl "cat /etc/lsb-release && swift -version && swift test -Xswiftc -warnings-as-errors $${SANITIZER_ARG-}"
```

Comment:
debug commands intentional?

Reply:
yes, we need this everywhere, otherwise it's impossible to tell what version of Swift (especially for `master` snapshots) was actually run.

Reply:
(same for the OS version, better to have that in there 100% clear)